app/models/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
+  primary_abstract_class
+end
```

Comment:
- [ ] Please add `current_user` method to  return the first user from the database

![image](https://user-images.githubusercontent.com/75126481/156565113-ef586cbd-714d-47f6-8ead-c5fa5713726b.png)

It is going to look like this :point_down: 

```ruby
def current_user
  User.first
end
```

Reply:
resolved